src/assets/scss/_gift-config.scss

```diff
@@ -31,7 +31,7 @@ label.custom-amount {
 .checkbox {
   label {
     min-height: 29px;
-    padding-left: 0px;
+    padding-left: 20px;
```

Comment:
What is the reason for this change on top of the other? I think the scope of the issue is only in saved payment methods.